oap-server/ai-pipeline/pom.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
+  ~ contributor license agreements.  See the NOTICE file distributed with
+  ~ this work for additional information regarding copyright ownership.
+  ~ The ASF licenses this file to You under the Apache License, Version 2.0
+  ~ (the "License"); you may not use this file except in compliance with
+  ~ the License.  You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.skywalking</groupId>
+        <artifactId>oap-server</artifactId>
+        <version>9.5.0-SNAPSHOT</version>
+    </parent>
+
+    <groupId>org.apache.skywalking</groupId>
+    <artifactId>ai-pipeline</artifactId>
+
+    <description>
+        AI pipeline module include pipelines connecting the OAP kernel and any 3rd party AI and/or algorithm
+        server. The kernel would call the pipeline in the specific context/scenario with well formatted telemetry data,
+        and expect the remote to respond with the result of the AI/algorithm as quick as possible, but in async way in
+        case of large size of data or lower performance behavior of the remote.
```

Comment:
@jcchavezs This is the responsibility of the pipeline. Bundle the logic of assembling AI/algorithm required data into OAP kernel, and provide a clear path for those data.